internal/vppinit/vppinit.go

```diff
@@ -76,6 +80,11 @@ func LinkToAfPacket(ctx context.Context, vppConn api.Connection, tunnelIP net.IP
 		WithField("duration", time.Since(now)).
 		WithField("vppapi", "SwInterfaceSetFlags").Debug("completed")
 
+	err = addArp(ctx, vppConn, swIfIndex)
```

Comment:
Please call this addIPNeighbor - we want to cover more than just arp :)

Reply:
Yep, fixed